### ISSUE TITLE
Splash image does not appear properly on Android 12

### ISSUE THREAD

**Issue by cmeuser** [jonbhanson/flutter_native_splash#374]
**Describe the bug**
Splash image does not fill screen on Android 12 devices.  The splash image appears as a small square in the center of the screen.  On Android 11, and prior, the splash screen fills the screen appearing as expected.  Here is my configuration:

flutter_native_splash:
  color: '00182e'
  image: assets/images/splash.png
  android_12:
    color: '00182e'
    image: assets/images/splash.png
  ios: false
  android: true
  android_gravity: fill

**To Reproduce**

Github project is accessible here:
https://github.com/cmeuser/splash_demo

**Expected behavior**
I expect that the splash image will fill the screen.

**Screenshots**

**Device (please complete the following information):**
Samsung 21 Ultra (Android 12)
Pixel 6 Pro (Android 12)

**Flutter doctor**
Please paste the results of the `flutter doctor -v` here

MVVKV3FQJY:splash_demo meusercx$ flutter doctor -v
[✓] Flutter (Channel stable, 3.0.2, on macOS 12.3.1 21E258 darwin-arm, locale en-US)
    • Flutter version 3.0.2 at /Users/meusercx/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision cd41fdd495 (2 weeks ago), 2022-06-08 09:52:13 -0700
    • Engine revision f15f824b57
    • Dart version 2.17.3
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at /Users/meusercx/Library/Android/sdk
    • Platform android-33, build-tools 32.0.0
    • ANDROID_SDK_ROOT = /Users/meusercx/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)

[✓] VS Code (version 1.68.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.42.0

[✓] Connected device (5 available)
    • Pixel 6 Pro (mobile)      • 1C301FDEE0077P • android-arm64  • Android 12 (API 32)
    • SM G998U1 (mobile)        • R5CR415VF0Y    • android-arm64  • Android 12 (API 31)
    • sdk gphone arm64 (mobile) • emulator-5554  • android-arm64  • Android 11 (API 30) (emulator)
    • macOS (desktop)           • macos          • darwin-arm64   • macOS 12.3.1 21E258 darwin-arm
    • Chrome (web)              • chrome         • web-javascript • Google Chrome 103.0.5060.53

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

**Additional context**

**Comment by jonbhanson** [comment]
I ran your demo and it worked for me.  Did you read [this note](https://pub.dev/packages/flutter_native_splash#android-12-support) in the readme:

> PLEASE NOTE: The splash screen may not appear when you launch the app from Android Studio. However, it should appear when you launch by clicking on the launch icon in Android.

**Comment by ForeverCarousel** [comment]
me too, but my phone is not origin android launcher, it's rand's own launcher,  I don't have an origin system android phone,so I'm not sure if that's the reason.
But strangely it works on another Android 11 device, even though that device is also a 3rd party launcher

**Comment by jonbhanson** [comment]
> The problem I am seeing is that the image does not fill the screen on Android 12 devices. 

Yes, this is how splash screens work on Android 12.  [Full docs](https://developer.android.com/guide/topics/ui/splash-screen).

**Comment by cmeuser** [comment]
> > The problem I am seeing is that the image does not fill the screen on Android 12 devices.
> 
> Yes, this is how splash screens work on Android 12. [Full docs](https://developer.android.com/guide/topics/ui/splash-screen).

I see.  Thank you.
